i6_models/parts/conformer/convolution.py

```diff
@@ -0,0 +1,80 @@
+from __future__ import annotations
+
+from dataclasses import dataclass
+
+import torch
+from torch import nn
+from i6_models.config import ModelConfiguration
+from typing import Callable, Union, Any, Type
+
+
+@dataclass
+class ConformerConvolutionV1Config(ModelConfiguration):
+    channels: int
+    """number of channels for conv layers"""
+    kernel_size: int
+    """kernel size of conv layers"""
+    dropout: float
+    """dropout probability"""
+    activation: Union[nn.Module, Callable[[torch.Tensor], torch.Tensor]]
+    """activation function applied after norm"""
+    norm: Union[nn.BatchNorm1d, nn.LayerNorm]
```

Comment:
Why only those two types? Why can't you accept any module here? And maybe even `Callable`?